ng2ts/tests/templateAddActionTest.ts

```diff
@@ -26,12 +26,14 @@ describe('Blabla', () => {
 
   it(`app.component.ts: Add a 'search' method on the component, that takes a 'searchString' parameter.`, () => {
     const fixture = TestBed.createComponent(AppComponent);
+    const paramName = fixture.componentInstance.search.toString();
     fixture.detectChanges();
     chai.expect(fixture.componentInstance.search, `Search should be a function`).is.a('function');
     chai.expect(fixture.componentInstance.search.length, `Search should take one parameter`).equals(1);
+    chai.expect(paramName.toString().slice(paramName.indexOf('(') + 1, paramName.indexOf(')'))).equals('searchString');
```

Comment:
Not familiar with chai, but can't you do something like `expect(paramName).toContain('searchString')`?